recipes/holospy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "holospy" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holospy-{{ version }}.tar.gz
+  sha256: 23ec13636bcdabccfceab6a4103bd05f3a04df32ea37e43997c30ec745a6b48d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=64
+    - setuptools-scm >=8
+    - pip
+  run:
+    - python >=3.8
+    - hyperspy-base >=2.0
+    - numpy >=1.20.0
+    - scipy >=1.5.0
+
+test:
+  imports:
+    - holospy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Analysis of (off-axis) holography data with HyperSpy.
+  home: https://github.com/hyperspy/holospy
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  doc_url: https://hyperspy.org/holospy
+  dev_url: https://github.com/hyperspy/holospy
+
+extra:
+  recipe-maintainers:
+    - CSSFrancis
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```

Comment:
Same as above, @CSSFrancis, @francisco-dlp, @jlaehne, can you confirm that you are willing to be maintainers?